app/routes/admin-panel/global-system-message.js

```diff
@@ -7,12 +7,15 @@ export default class AdminPanelGlobalSystemMessageRoute extends Route {
   @service session;
   @service router;
   @service('globalSystemMessage') messageService;
+  @service impersonation;
 
   beforeModel(transition) {
     this.session.requireAuthentication(transition, 'login');
-
-    if (!this.currentSession.showAdminFeatures) {
```

Comment:
i sortof liked having this as a simple getter. don't we want to keep this?

Reply:
Indeed alot easier in use